sound/audioclass.py

```diff
@@ -2,11 +2,14 @@
 from collections import deque
 import rospy
 import math
+import numpy as np
```

Comment:
as far as I can tell, numpy is not used anywhere, so this import is not needed.

I have just started merging this by hand, and so will clean this up.  However, in the future, don't import modules you don't use.